src/http/endpoints/update_profile.rs

```diff
@@ -19,11 +19,12 @@ struct ProfileLinkUpdate {
 struct ProfileUpdate {
     pub display_name: Option<String>,
     pub description: Option<String>,
-    pub workplace: Option<String>,
+    pub organization: Option<String>,
+    pub location: Option<String>,
     pub links: Vec<ProfileLinkUpdate>,
 }
 
-#[post("/v1/entity/{entity}")]
+#[patch("/v1/entity/{entity}")]
```

Comment:
I was like "why does the client still work" and then I realized we haven't updated the client or covered the endpoint with end-to-end tests. Whoopsie! :)

Reply:
Hahaha, you're right. Should I create an ayb client command for getting and updating the profile? Or should I just send requests to the endpoint in the E2E tests? :)

Reply:
Great question! While it's a little clunky, an `ayb` client command would be great. Not to be too prescriptive (you can decide as you play with it), but something like

`ayb client update_profile sofiaritz links https://link.one https://link.two`
(links is the hardest, but `organization`, `location`, etc. would be similar but with only one arg in the list)

Reply:
...and I guess something similar for `get_profile?` (I remember a discussion when you introduced the details enpoint that you wanted to keep the database list and the other profile information separate so table/csv export would still work, but I don't feel strongly either way.)